tc_aws/aws/bucket.py

```diff
@@ -121,7 +125,9 @@ async def delete(self, path):
             Key=self._clean_key(path),
         )
 
-    def _clean_key(self, path):
+    @staticmethod
+    @lru_cache()
```

Comment:
what's that for?

Reply:
A very minor performance change.  Nothing important really.